superduperdb/backends/dask/compute.py

```diff
@@ -23,14 +24,12 @@ def __init__(
         serializers: t.Optional[t.Sequence[t.Callable]] = None,
         deserializers: t.Optional[t.Sequence[t.Callable]] = None,
         local: bool = False,
-        envs: t.Optional[t.Dict[str, t.Any]] = None,
         **kwargs,
     ):
-        envs = envs or {}
         self.futures_collection: t.Dict[str, distributed.Future] = {}
 
         if local:
-            cluster = distributed.LocalCluster(env=envs)
+            cluster = distributed.LocalCluster(processes=False)
```

Comment:
u can remove entire line
and keep
 self.client = distributed.Client(processes=False)

Im not sure if processes is argument for LocalCluster

https://docs.dask.org/en/stable/deploying-python.html

Reply:
Both lines are needed.
The first one creates a local dask cluster, and the second one is for connecting to it.

Reply:
Actually 
client = distributed.Client(processes=False) is short for 

```
cluster = distributed.LocalCluster(processes=False)
distributed.Client(cluster)
```


can you please check this official dask https://docs.dask.org/en/stable/deploying-python.html
its clearly stated there